pymeasure/instruments/pendulum/cnt91.py

```diff
@@ -0,0 +1,191 @@
+        return super().ask(command).rstrip("\n")
+
+    @property
+    def operation_complete(self):
+        """Is True if operation is complete."""
+        return self.ask("*OPC?") == "1"
+
+    @property
+    def batch_size(self):
+        """Maximum number of buffer entries that can be transmitted at once."""
+        if not hasattr(self, "_batch_size"):
+            self._batch_size = int(self.ask("FORM:SMAX?"))
+        return self._batch_size
+
+    def read_buffer(self):
+        """
+        Read out the entire buffer.
+
+        :return : Frequency values from the buffer.
+        """
+        while not self.operation_complete:
+            # Wait until the buffer is filled.
+            sleep(0.01)
+        data = []
+        # Loop until the buffer is completely read out.
+        while True:
+            # Get maximum number of buffer values.
+            new = self.values(":FETC:ARR? MAX")
+            data += new
+            # Last values have been read from buffer.
+            if len(new) < self.batch_size:
+                break
+        return data
+
+    external_start_arming_source = Instrument.control(
+        "ARM:SOUR?",
+        "ARM:SOUR %s",
+        (
+            "Select arming input or switch off the start arming function."
+            "Options are 'A', 'B' and 'E' (rear). 'IMM' turns trigger off."
+        ),
+        validator=strict_discrete_set,
+        values={"A": "EXT1", "B": "EXT2", "E": "EXT4", "IMM": "IMM"},
+        map_values=True,
+    )
+
+    external_arming_start_slope = Instrument.control(
+        "ARM:SLOP?",
+        "ARM:SLOP %s",
+        "Set slope for the start arming condition.",
+        validator=strict_discrete_set,
+        values=["POS", "NEG"],
+    )
+
+    continuous = Instrument.control(
+        "INIT:CONT?",
+        "INIT:CONT %s",
+        "Turn continuous measurement 'ON' or 'OFF'",
+        strict_discrete_set,
+        values=["ON", "OFF"],
+    )
+
+    measurement_time = Instrument.control(
+        ":ACQ:APER?",
+        ":ACQ:APER %f",
+        "Gate time for one measurement in s.",
+        validator=strict_range,
+        values=[2e-9, 1000],  # Programmer's guide 8-92
+    )
+
+    format = Instrument.control(
+        "FORM?",
+        "FORM %s",
+        "Reponse format (ASCII or REAL).",
+        validator=strict_discrete_set,
+        values=["ASCII", "REAL"],
+    )
+
+    interpolator_calibration = Instrument.control(
+        ":CAL:INT:AUTO?",
+        "CAL:INT:AUTO %s",
+        "Controls if interpolators should be calibrated automatically.",
+        strict_discrete_set,
+        values=["ON", "OFF"],
+        get_process=lambda v: "ON" if v else "OFF",
+    )
+
+    def configure_frequency_array_measurement(self, n_samples, channel):
+        """
+        Configure the counter for an array of measurements.
+
+        :param n_samples : The number of samples
+        :param channel : Measurment channel (A, B, C, E, INTREF)
```

Comment:
trailing  space in  `:param n_samples :` breaks the formatting.

Reply:
Fixed in 57ef117 also in some other instances.